cfme/common/vm_views.py

```diff
@@ -64,6 +64,40 @@ def InstanceEntity():  # noqa
     })
 
 
+class NonEditableTable(Table):
+    """Wigdet for non-editable table. used for selecting value"""
+    def fill(self, values):
+        """Clicks on item - fill by selecting required value"""
+        value = values.get('name', '<None>')
+        changed = False
+        if value != self.currently_selected:
+            changed = True
+            self.row(name=value).click()
+            return changed
```

Comment:
**Required**

Collapse these two return statements, no need for the `else` splitting them.